images/syncer/Dockerfile

```diff
@@ -17,7 +17,7 @@
 ARG GOLANG_IMAGE=golang:1.13
 
 # This build arg allows the specification of a custom base image.
-ARG BASE_IMAGE=photon:3.0
+ARG BASE_IMAGE=gcr.io/cloud-provider-vsphere/extra/csi-driver-base:latest
 
 ################################################################################
 ##                              BUILD STAGE                                   ##
```

Comment:
Can we club the `COPY` commands into a single command? Each `COPY` command creates a layer in the image.

Reply:
for each `COPY` command destination directory is different so I can not club into a single command.

```
COPY go.mod go.sum ./
COPY pkg/    pkg/
COPY cmd/    cmd/
```

also, they are present in the builder image and not in the final image.
in the syncer image we have just one copy command.

```
COPY --from=builder /build/vsphere-syncer /bin/vsphere-syncer
```

Reply:
Can we not do something like this?

`COPY go.mod go.sum pkg cmd ./`

Reply:
no destination directories are different.

Reply:
ignore my comment. let me try. but as I said, this is in the intermediate temporary builder image. will not benefit the final image.

Reply:
@SandeepPissay `COPY go.mod go.sum pkg/ cmd/ ./` did not help.

```
Step 7/16 : COPY go.mod go.sum pkg/ cmd/ ./
```
build ./cmd/vsphere-csi: cannot find module for path ./cmd/vsphere-csi

Reply:
the destination directories need to be specified, so we can not club them into a single command.